js/src/main/scala/fm/common/rich/RichNode.scala

```diff
@@ -29,16 +26,6 @@ final class RichNode[T <: Node](val self: T) extends AnyVal {
   /** Detatch this node from the DOM */
   def detatch(): T = if (hasParent) self.parentNode.removeChild(self).asInstanceOf[T] else self
   
-  /** Find the closest ancestor of the current element (or the current element itself) that matches a class. */
-  def closest[A <: Node : ClassTag]: A = closestImpl(self)
```

Comment:
What's wrong with this implementation?

Reply:
the lack of the selector, I was having an implicit conflict if this was still around vs having one with a selector also as an implicit . I can see if I can isolate the specific issue